internal/contour/listener_test.go

```diff
@@ -135,14 +136,14 @@ func TestRecomputeListener(t *testing.T) {
 
 func TestRecomputeTLSListener(t *testing.T) {
 	ingresss_http := listener(ENVOY_HTTPS_LISTENER, "0.0.0.0", 8443)
-	ingresss_http.FilterChains = []*v2.FilterChain{{
-		Filters: []*v2.Filter{
+	ingresss_http.FilterChains = []envoy_api_v2_listener.FilterChain{{
```

Comment:
I think this unintentional change from a slice of pointers to a slice of values has caused a lot of unnecessary churn.